docs/docs-components/PropTable.js

```diff
@@ -69,28 +99,8 @@ function Td({
   );
 }
 
-function isNumeric(value) {
-  return /^-?\d+$/.test(value);
-}
-
-const transformDefaultValue = (input) => {
-  if (input === 'true') {
-    return true;
-  }
-  if (input === 'false') {
-    return false;
-  }
-  if (typeof input === 'string' && isNumeric(input)) {
-    return parseFloat(input);
-  }
-  return input;
-};
-
-const sortBy = (list, fn) => [...list].sort((a, b) => fn(a).localeCompare(fn(b)));
-
 type Props = {|
-  // $FlowFixMe[unclear-type]
-  Component?: ComponentType<any>,
+  componentName: string,
```

Comment:
We previously weren't really doing anything with Component, but we need the component name for this feature (and a better accessibility label)